universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -198,6 +227,52 @@ private ContainerTag renderBlockList(
     return container;
   }
 
+  private ContainerTag blockMoveButtons(
+      Request request,
+      long programId,
+      ImmutableList<BlockDefinition> blockDefinitions,
+      BlockDefinition blockDefinition) {
+    String moveUpFormAction =
+        routes.AdminProgramBlocksController.move(programId, blockDefinition.id()).url();
+    // Move up button is invisible for the first block
+    String moveUpStyles =
+        StyleUtils.joinStyles(
+            AdminStyles.MOVE_BLOCK_BUTTON,
+            (blockDefinition.id() == blockDefinitions.get(0).id()) ? Styles.INVISIBLE : "");
```

Comment:
FYI: There is the `first:` pseudo selector available in Tailwind that could help achieve unique first-child element behavior: https://tailwindcss.com/docs/hover-focus-and-other-states#first-child. However, you may need to add something to `tailwind.config.js` in order to make this work (ping me if you need help). This is non-blocking IMO, but it may be a cleaner way to implement this.

Reply:
Can't really use tailwind's `first` because of how deeply nested the element we want to make invisible is.